ThreadGroup/Controllers/TGMainViewController.m

```diff
@@ -87,6 +87,9 @@ @interface TGMainViewController() <TGDeviceStepViewDelegate, TGSelectDeviceStepV
 //Thread Network Config
 @property (strong, nonatomic) TGNetworkConfigViewController *threadConfig;
 
+//Cancelling Router Connections
+@property (nonatomic) BOOL shouldCancelRouterConnection;
```

Comment:
perhaps renaming to shouldIgnoreRouterConnection, since we aren't actually canceling the connection, we are just ignoring the callback. @ButkiewiczP  thoughts?